src/app/components/MatchCardSitter/MatchCardSitter.tsx

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import styles from "./MatchCardSitter.module.css";
+
+export type MatchCardSitterProps = {
+  imgSrc: string;
+  name: string;
+  experience: string;
+  mail: string;
+  phone: string;
+};
+
+function MatchCardSitter({
+  imgSrc,
+  name,
+  experience,
+  mail,
+  phone,
+}: MatchCardSitterProps): JSX.Element {
+  return (
+    <div className={styles.container}>
+      <img className={styles.matchImage} src={imgSrc} alt="Profile Image" />
+      <div className={styles.matchInfo}>
+        <span className={styles.matchName}>{name}</span>
+        <span className={styles.matchAge}>{experience}</span>
+        <span className={styles.matchMail}>{mail}</span>
+        <span className={styles.matchPhone}>{phone}</span>
+      </div>
+    </div>
+  );
+}
+
+export default MatchCardSitter;
```

Comment:
This is very similar to `MatchCardDoc`. 
You can reduce it to a single component.
Is `experience` / `age` the only difference?
Just call it `subtitle` or make use of a composition.

Reply:
I will fetch that info (experience/age) from my database though.. can I still fetch different information if I use the same component? 
What do you mean by composition?